webapp/sdf/src/components/list-component/list-component.spec.js

```diff
@@ -0,0 +1,141 @@
+                    component.findComponent(ItemComponent).vm.$emit('updateEvent', new Item(1, "Eintrag1", 6))
+                    await component.vm.$nextTick()
+                    expect(component.vm.$data.itemList.some(item =>
+                        item.id == 1 &&
+                        item.title =="Eintrag1" &&
+                        item.votes == 5
+                    )).toBe(false)
+                })
+                
+                it('adds new item 1', async () => {
+                    component.findComponent(ItemComponent).vm.$emit('updateEvent', new Item(1, "Eintrag1", 6))
+                    await component.vm.$nextTick()
+                    expect(component.vm.$data.itemList.some(item =>
+                        item.id == 1 &&
+                        item.title =="Eintrag1" &&
+                        item.votes == 6
+                    )).toBe(true)
+                })
+                
+                it('does not remove item 2', async () => {
+                    component.findComponent(ItemComponent).vm.$emit('updateEvent', new Item(1, "Eintrag1", 6))
+                    await component.vm.$nextTick()
+                    expect(component.vm.$data.itemList.some(item =>
+                        item.id == 2 &&
+                        item.title =="Eintrag2" &&
+                        item.votes == 2
+                    )).toBe(true)
+                })
+            
+                it('does not remove item 3', async () => {
+                    component.findComponent(ItemComponent).vm.$emit('updateEvent', new Item(1, "Eintrag1", 6))
+                    await component.vm.$nextTick()
+                    expect(component.vm.$data.itemList.some(item =>
+                        item.id == 3 &&
+                        item.title =="Eintrag3" &&
+                        item.votes == -2
+                    )).toBe(true)
+                })
+            })
+
+            describe('toggle order button clicked', () => {
+                it('was ascending, now descending', async () => {
+                    const toggleButton = component.find('.toggle-btn')
+                    toggleButton.trigger('click')
+                    await component.vm.$nextTick()
+                    const firstElement = component.findComponent(ItemComponent)
+                    expect(firstElement.text()).toContain('Eintrag3')
+                })
+
+                it('was descending, now ascending', async () => {
+                    const toggleButton = component.find('.toggle-btn')
+                    toggleButton.trigger('click')
+                    await component.vm.$nextTick()
+                    toggleButton.trigger('click')
+                    await component.vm.$nextTick()
+                    const firstElement = component.findComponent(ItemComponent)
+                    expect(firstElement.text()).toContain('Eintrag1')
+                })
+            })
+        })
+    })
+})
```

Comment:
Just clean code improvements.
```suggestion
})